docroot/modules/custom/mass_map/src/MapLocationFetcher.php

```diff
@@ -230,6 +230,7 @@ public function getLocations(array $nids) {
         'href' => 'https://www.google.com/maps/place/' . $locations['imagePromos']['items'][$key]['location']['text'],
         'type' => "external",
         'info' => '',
+        'labelContext' => t('to @location', ['@location' => $locations['imagePromos']['items'][$key]['location']['text']]),
```

Comment:
@arthurbaghdas  This prints the address for the labelContext instead of the location name.  Can you make it to the location name?
<img width="701" alt="screenshot" src="https://user-images.githubusercontent.com/9633303/115413775-75ae8b80-a1c3-11eb-8d4b-3f342e9b9b70.png">

Reply:
@ygannett please take a look again.

Reply:
@arthurbaghdas This time, no `span.ma__visually-hidden` is rendered. Please check.
<img width="771" alt="direction-context" src="https://user-images.githubusercontent.com/9633303/115610133-73792980-a2b6-11eb-9a4d-f3edd0adddd4.png">

Reply:
<img width="785" alt="Screen Shot 2021-04-26 at 15 48 38" src="https://user-images.githubusercontent.com/62405127/116077981-22e34100-a6a7-11eb-9493-d6a60f78f12c.png">

I can see it rendering both logged in and anonymous. @ygannett